spec/support/webmock_ruby24_bridge.rb

```diff
@@ -0,0 +1,10 @@
+# Ruby 2.4.0 removed the closed? check in the conditional in: s.close if !s.closed?
+# Webmock was changed to add close to StubSocket along with another change..
+# https://github.com/ruby/ruby/commit/f845a9ef76c0195254ded79c85c24332534f4057
+# https://github.com/bblimke/webmock/commit/8f2176a1fa75374df55b87d782e08ded673a75b4
+# We should upgrade webmock but that requires some re-recording of cassettes (I think)
+# and maybe other things.
+class StubSocket
+  def close
+  end
+end
```

Comment:
We can either upgrade webmock or do this for now.  I tried upgrading it locally and it looks at the very least that we'd need to re-record our vcr cassettes.  I'm not sure if that's all, so I went the easy route here in hopes we can upgrade in a followup PR.